assets/sass/components/global/_googlesitekit-help-menu-link.scss

```diff
@@ -0,0 +1,22 @@
+/**
+ * HelpMenuLink styles.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin .googlesitekit-help-menu-link .googlesitekit-cta-link {
+	color: $c-nearblack;
+	font-size: 16px;
```

Comment:
It would be great to use a variable for the size instead of the fixed value. It looks like our definitions in `$typography` are the only place it is defined but I'm not sure if the `responsive-text` mixin is appropriate here as I don't see it used on the other menus/items. I suppose it's okay to keep as is; what do you think?

Reply:
I thought about using a variable too, but as this is from newer designs I don't know what to call this. I think the newer design mockups are using slightly different sizes, so we are stuck in the "in-between" state.

I don't see this size being used elsewhere or clearly defined yet, so I can't think of a useful variable to assign it to. I'd prefer to, but I think for now this is fine. We can tweak it later though 👍🏻